src/main/java/com/artipie/api/RestApi.java

```diff
@@ -71,6 +72,11 @@ public final class RestApi extends AbstractVerticle {
      */
     private final Optional<KeyStore> keystore;
 
+    /**
+     * Artipie settings.
+     */
+    private final Settings settings;
```

Comment:
I do not think we should have `Settings` as a field here, as it's not really required, layout is already in the class fields